### ISSUE TITLE
S1 - SCREEN READER OPTIONS

### ISSUE THREAD

**Issue by prodb** [pace-running/pace3#13]
Business Context:
In order to enable users with visual impairments (e.g. with dyslexia) to perceive the information effectively, a screen reader function should be presented clearly on every page that reads the content aloud to the user.

Research Objective:
The goal of this spike is to find out screen reader options for the website that fit in the given tech stack.
Aspects to consider:
- Voice (gender, tone)
- Compatibility
- Performance
